### PR TITLE
update rules to match production

### PR DESCRIPTION
**Pull request by andrewkrug**

### CODE REVIEW

File: rules/gcp-gsuite-SAML-claims.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+function (user, context, callback) {
```
Comment (gdestuynder) [inline rules/gcp-gsuite-SAML-claims.js]: @andrewkrug is this duplicating https://github.com/mozilla-iam/auth0-deploy/blob/master/rules/SAML-test-mozilla-com-google.js ?

File: rules/SAML-AWS-mozillaiam-account-readonly.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+function (user, context, callback) {
+  var CLIENT_IDS = [
```
Comment (gdestuynder) [inline rules/SAML-AWS-mozillaiam-account-readonly.js]: faster would be   `if (context.clientID !== '2lTZtlpqx4bslng167l1BBqTMusCAJkZ') return callback(null, user, context); // AWS Federated Access for AWS IAM`

File: rules/gcp-gsuite-SAML-claims.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+    "enabled": true,
+    "order": 20
```
Comment (gdestuynder) [inline rules/gcp-gsuite-SAML-claims.json]: this is already taken, please use `30` f.e.

File: rules/gcp-gsuite-SAML-claims.js (hunk 1 above)
Comment (andrewkrug) [reply rules/gcp-gsuite-SAML-claims.js]: It does the same replace.  But for a different namespace / client_id.

File: rules/gcp-gsuite-SAML-claims.json (hunk 3 above)
Comment (andrewkrug) [reply rules/gcp-gsuite-SAML-claims.json]: fixed

**Pull request merged**